test/index.js

```diff
@@ -37,6 +37,37 @@ describe('Import type test', function() {
     expect(result.css.toString()).to.eql(EXPECTATION);
   });
 
+  it('escapes strings when escapeStrings option is set', function() {
+    const escapeStringsExpectation = `#results > li:before {
+  color: red;
+  content: "sony"; }
+
+#results2 > li:before {
+  color: red;
+  content: "sony"; }
+`
+    let result = sass.renderSync({
+      file: './test/fixtures/escape-strings/style.scss',
+      escapeStrings: true,
+      importer: jsonImporter
+    });
+
+    expect(result.css.toString()).to.eql(escapeStringsExpectation);
+  });
+
+  it(`throws on complex JSON when escapeStrings option is not set`, function() {
```

Comment:
I think "complex JSON" isn't specific enough. You could construct complex JSON that doesn't throw in this test. Something like "throws when parsing a string that isn't a valid SASS expression".